[PATHS]
DATA_FILE_PATH = ./tests/resources/
PDF_DATA_FILE_NAME = PDF_FILE.pdf
PDF_CONTENT_REFERENCE = pdf_content_reference.txt
PDF_METADATA_REFERENCE = pdf_metadata_reference.json
PDF_URI = http://arxiv.org/pdf/cs/9308101v1
NOT_FOUND_PDF_URI = http://arxiv.org/pdf/cs/cvzqcdcsdcdsfcdfvdfvdfvfdvdv
NOT_PDF_URI = https://docs.python.org/3/tutorial/errors.html
WRONG_PDF_URI = ?!
FEED_DATA_FILE_NAME = feed_reference.txt
REFERENCE_PDF_URIS_FILE_NAME = pdf_uris_reference.txt

[ArXivParser]
time_step = 604800
arxiv_url = https://export.arxiv.org/api/
cat = cs.ai
max_results = 1000

[Cooldown Manager]
cooldown_manager_uri = http://172.17.0.2:5000/

[tool:pytest]
testpaths = tests

[coverage:run]
branch = True
source = .